### ISSUE TITLE
Android - Accessibility - Language Screen - Poor Color Contrast of Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#202]
The attached screenshot shows the black text over the image has poor color contrast and needs to be fixed. Some of the language buttons might have issues, too, being on darker parts of the image.

I assume this will be corrected when the application's look changes back to what it used to be in one of the next builds.

**Comment by jfc3-dol** [comment]
@pbhatt17 
I tested the language page on Android, and the color contrast it back to what it used to be and again passes.

So I'm going to close this issue.
